committeeoversightapp/views.py

```diff
@@ -104,14 +105,15 @@ def filter_queryset(self, qs):
 
     def prepare_results(self, qs):
         json_data = []
+        detail_string = "<a href=\"/hearing/view/{0}\">{1}</a>"
```

Comment:
What's your thinking here behind hardcoding the link instead of using `reverse()` to generate it on the fly?

Reply:
I'll use reverse!